doc/source/reference/migration_guide_from_1.x_to_2.0.rst

```diff
@@ -0,0 +1,117 @@
+This is a guide for people moving from NetworkX 1.X to NetworkX 2.0
+
+As you know we have made some major changes to the methods in the Multi/Di/Graph classes.
+Methods that used to return lists are removed and the methods that returned iterators
+are renamed(removing the _iter suffix).
+For example we hade two methods to get nodes of a graph, ``G.nodes()`` and ``G.nodes_iter()``. ``G.nodes_iter()`` is renamed to ``G.nodes()``, hence 
+``G.nodes()`` returns an iterator now instead of list.
+
+The methods changed are explained with examples below.
+
+	>>> import networkx as nx
+	>>> G = nx.complete_graph(5) 
+	>>> G.nodes()
+	<dictionary-keyiterator at 0x102ae8730>
+
+
+People suprised by this output and expecting something like this
+
+	>>> G.nodes()
+	[0, 1, 2, 3, 4]
+
+don't panic. Nothing is wrong.
+
+Now ``G.nodes()`` returns an iterator. If you don't like this you can always get the old output by
```

Comment:
Instead of "don't like this", use "If you want a list of nodes send the iterator to the Python list function"